digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1174,6 +1179,33 @@ val userNotificationService: UserNotificationService = Digiroad2Context.userNoti
     }
   }
 
+  get("/manoeuvre/inaccurates") {
+    val user = userProvider.getCurrentUser()
+    val municipalityCode = user.configuration.authorizedMunicipalities
+    municipalityCode.foreach(validateUserMunicipalityAccessByMunicipality(user))
+
+    user.isOperator() match {
+      case true =>
+        manoeuvreService.getInaccurateRecords(Manoeuvres.typeId)
```

Comment:
Does the ` manoeuvreService.getInaccurateRecords(Manoeuvres.typeId)` need to receive the type id, since the method is inside the manoeuvreService?